src/entrypoints/localApp.ts

```diff
@@ -0,0 +1,74 @@
+import { nextGenParse } from '../commands/src/shared/next-gen-parse';
+import { nextGenHtml } from '../commands/src/shared/next-gen-html';
+import { prepareBuildAndGetDependencies } from '../commands/src/helpers/dependency-helpers';
+import { nextGenStage } from '../commands/src/shared/next-gen-stage';
+import { oasPageBuild } from '../commands/src/shared/oas-page-build';
+import { persistenceModule } from '../commands/src/shared/persistence-module';
+import { nextGenDeploy } from '../commands/src/shared/next-gen-deploy';
+
+async function localApp() {
+  // TODO: Fetch this from repos_branches
```

Comment:
The `localApp.ts` is used by the `Dockerfile.local`. Created this to test and verify that the commands are working as expected.